version/version.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2015 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package version
+
+import "fmt"
+
+var (
+	// RELEASE returns the release version
+	RELEASE = "UNKNOWN"
+	// REPO returns the git repository URL
+	REPO = "UNKNOWN"
+	// COMMIT returns the short sha from git
+	COMMIT = "UNKNOWN"
+)
+
+// String returns information about the release.
+func String() string {
+	return fmt.Sprintf(`-------------------------------------------------------------------------------
```

Comment:
Actually i do not like printing version info like this in un structure format which is very difficult to parse for users if needed.

Another good example to do this job is what kubernetes does for kubelet which is printing version info in json format.

Reply:
I'd prefer a structured format as well

Reply:
A simple but useful way to achieve this goal is to define a struct and logout the json output of the version instance.